lib/ansible/modules/uri.py

```diff
@@ -14,6 +14,7 @@
   - Interacts with HTTP and HTTPS web services and supports Digest, Basic and WSSE
     HTTP authentication mechanisms.
   - For Windows targets, use the M(ansible.windows.win_uri) module instead.
+  - Be Aware, the used remote module (ansible.legacy.uri) does not support check mode (-C --check)
```

Comment:
That is not the correct FQCN, this module is `ansible.builtin.uri` (`ansible.legacy` is a special namespace that can resolve to local overrides instead of the builtin.) This is also an unnecessary note; https://github.com/ansible/ansible/blob/584ee3b7f788f143d7a8ea22e9ae9c6f454c100a/lib/ansible/modules/uri.py#L238-L239 is the standardized way of documenting whether a module works in check mode.